package.json

```diff
@@ -117,7 +117,7 @@
     "cover": "istanbul cover node_modules/mocha/bin/_mocha -- ./test/unit/index.js"
   },
   "dependencies": {
-    "@twilio/webrtc": "github:twilio/twilio-webrtc.js#4.1.0-rc1",
+    "@twilio/webrtc": "github:twilio/twilio-webrtc.js#87f7e61106f58e29364999d8753c773e5602536f",
```

Comment:
is this temporary right ?

Reply:
@makarandp0 Yes. Before I release the next RC, I will also release the corresponding twilio-webrtc.js RC and update package.json.